src/redux/reducers/rootReduces.ts

```diff
@@ -0,0 +1,28 @@
+import { TReduxRootState } from "types";
+import initState from "../state/initialState";
+
+const rootReduces = (state: TReduxRootState = initState, action: any) => {
```

Comment:
The `any` type in TypeScript is a dangerous "escape hatch" from the type system. Using `any` disables many type checking rules and is generally best used only as a last resort or when prototyping code.

```suggestion
const rootReduces = (state: TReduxRootState = initState, action: Action) => {
```